server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -150,16 +141,8 @@ public T call() throws Exception
 
   public <T> T retryTransaction(final TransactionCallback<T> callback, final int quietTries, final int maxTries)
   {
-    final Callable<T> call = new Callable<T>()
-    {
-      @Override
-      public T call() throws Exception
-      {
-        return getDBI().inTransaction(callback);
-      }
-    };
     try {
-      return RetryUtils.retry(call, shouldRetry, quietTries, maxTries, null);
+      return RetryUtils.retry(() -> getDBI().inTransaction(callback), shouldRetry, quietTries, maxTries, null, null);
```

Comment:
Shouldn't pass `null`? There is an overload

Reply:
Done.